### PR TITLE
Extract redic commands

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lib/reliable/queue.rb [hunk 1]
```diff
@@ -136,38 +123,20 @@ def process(uuid, item, &block)
       end
     rescue StandardError => e
       move uuid, processing.key, failed_processing.key
+      # TODO: if #move raises, what do we do?
```
Comment (rylev) [inline lib/reliable/queue.rb:126]: Can move raise?

Comment (rylev) [inline lib/reliable/queue.rb:126]: I guess with network issues only right?

**Comment by rylev** [comment]
LGTM :heart:

File: lib/reliable/queue.rb (hunk 1 above)
Comment (ghost) [inline lib/reliable/queue.rb:126]: Yeah. We should try to handle all the code paths I think.

**Pull request merged**